tests/platform_helper/test_command_check_cloudformation.py

```diff
@@ -51,7 +50,6 @@ def copilot_directory() -> Path:
 def application_under_test(copilot_directory):
     ensure_directory_does_not_exist(copilot_directory)
     os.chdir(copilot_directory.parent)
-    CliRunner().invoke(make_config)
```

Comment:
I'm fairly confident that this line actually didn't have any bearing on the tests below.